src/Entity/Field/ImageField.php

```diff
@@ -33,6 +33,11 @@ public function getValue(): array
         return $value;
     }
 
+    public function getTwigValue()
+    {
+        return $this->getValue();
```

Comment:
it should defaults to getFlattenedValue() somewhere inside Field class

Reply:
Good point. Have done so, and removed the overridden method from `Field\ImageField`.